src/data/mock/systemProfile.ts

```diff
@@ -0,0 +1,614 @@
+          objectID: '000-0000-0',
+          roleTypeID: '238-28-0',
+          assigneeType: 'PERSON',
+          assigneeUsername,
+          assigneeEmail,
+          assigneeOrgID: null,
+          assigneeOrgName: null,
+          assigneeFirstName,
+          assigneeLastName,
+          roleTypeName: 'System Maintainer',
+          roleID: '384-18958-0',
+          __typename: 'CedarRole'
+        },
+        {
+          application: 'alfabet',
+          objectID: '000-0000-0',
+          roleTypeID: '238-35-0',
+          assigneeType: 'PERSON',
+          assigneeUsername,
+          assigneeEmail: null,
+          assigneeOrgID: null,
+          assigneeOrgName: null,
+          assigneeFirstName,
+          assigneeLastName,
+          roleTypeName: "Contracting Officer's Representative (COR)",
+          roleID: '384-19359-0',
+          __typename: 'CedarRole'
+        },
+        {
+          application: 'alfabet',
+          objectID: '000-0000-0',
+          roleTypeID: '238-55-0',
+          assigneeType: 'PERSON',
+          assigneeUsername,
+          assigneeEmail,
+          assigneeOrgID: null,
+          assigneeOrgName: null,
+          assigneeFirstName,
+          assigneeLastName,
+          roleTypeName: 'QA Reviewer',
+          roleID: '384-19403-0',
+          __typename: 'CedarRole'
+        },
+        {
+          application: 'alfabet',
+          objectID: '000-0000-0',
+          roleTypeID: '238-56-0',
+          assigneeType: 'PERSON',
+          assigneeUsername,
+          assigneeEmail,
+          assigneeOrgID: null,
+          assigneeOrgName: null,
+          assigneeFirstName,
+          assigneeLastName,
+          roleTypeName: 'DA Reviewer',
+          roleID: '384-20087-0',
+          __typename: 'CedarRole'
+        }
+      ],
+      urls: [
+        {
+          id: '000-0000-1',
+          address: 'https://cms.gov',
+          isAPIEndpoint: false,
+          isBehindWebApplicationFirewall: true,
+          isVersionCodeRepository: false,
+          urlHostingEnv: 'Production',
+          __typename: 'CedarURL'
+        }
+      ],
+      systemMaintainerInformation: {
+        agileUsed: true,
+        deploymentFrequency: 'Bi-weekly',
+        devCompletionPercent: '75% - 99%',
+        devWorkDescription:
+          'Continued enhancements and improvements and alignment with legislative changes.  Also, working on an upcoming redesign effort for the website.',
+        netAccessibility:
+          'Accessible to both public internet and to CMS-internal network',
+        __typename: 'CedarSystemMaintainerInformation'
+      },
+      __typename: 'CedarSystemDetails'
+    }
+  }
+};
+
+export const query = {
+  request: {
+    query: GetSystemProfileQuery,
+    variables: {
+      cedarSystemId: '000-100-0'
+    }
+  },
+  result
+};
+
+export function getSystemProfileMockData(): SystemProfileData {
+  return getSystemProfileData(
+    result.data as GetSystemProfile
```

Comment:
It's possible to remove the type assertions and get by with just one non-null assertion; see https://github.com/CMSgov/easi-app/tree/EASI-2120/mock-data-type-cleanup. Declaring `result` with a type will make TS treat the `__typename` values as literal types instead of `string`s, the `PERSON` and `ORGANIZATION` string literals can be replaced by values of the `CedarAssigneeType` enum, and then you just need a non-null assertion on the result of calling `getSystemProfileData()`. Feel free to merge that branch; I ran it against CI [here](https://github.com/CMSgov/easi-app/actions/runs/2557650007), all tests passed.

Reply:
Glad to see how this is done done without the assertions. Will bring this in